mpitests/test_apply.py

```diff
@@ -0,0 +1,567 @@
+# Copyright 2018-2023 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for the :mod:`pennylane_lightning_gpu.LightningGPU` device (MPI).
+"""
+# pylint: disable=protected-access,cell-var-from-loop
+from mpi4py import MPI
+import math
+import numpy as np
+import pennylane as qml
+import pytest
+from pennylane import DeviceError
+
+try:
+    from pennylane_lightning_gpu.lightning_gpu import CPP_BINARY_AVAILABLE
+    from pennylane_lightning_gpu import LightningGPU
+    import pennylane_lightning_gpu as plg
+
+    if not CPP_BINARY_AVAILABLE:
+        raise ImportError("PennyLane-Lightning-GPU binary is not found on this platform")
+except (ImportError, ModuleNotFoundError):
+    pytest.skip(
+        "PennyLane-Lightning-GPU binary is not found on this platform. Skipping.",
+        allow_module_level=True,
+    )
+
+numQubits = 8
+
+def createRandomInitState(numWires):
```

Comment:
I don't know if we follow strict PEP8, but if so we should change this line and several others to use snake_case
```suggestion
def create_random_init_state(num_wires):
```

Reply:
done!